tools/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "plugins": [ "boa", "core" ],
+    "server": [ "daemon" ],
+    "pipboard": [ "pipboard" ]
+}
```

Comment:
The `server` and `pipboard` are not that necessary to be configurable, and it adds extra learn-cost, I would prefer this config is only for plugins directory just like:

```json
[ "packages/boa", "packages/core" ]
```

And rename it to a more specific one just like "dependencies.json".

Reply:
If `daemon` and `pipboard` are not inited, the logic for test `isLocal` will be left.
I think we shouldn't put test codes in to the user-run codes.

Reply:
I modified the format to:
```json
[
  {
    "destDir": "plugins",
    "dependencies": [ "boa", "core" ]
  },
  {
    "destDir": "server",
    "dependencies": [ "daemon" ]
  },
  {
    "destDir": "pipboard",
    "dependencies": [ "pipboard" ]
  }
]

```
This should make it more readable.

Reply:
@rickycao-qy @WenheLI May I ask you to review this design?

Reply:
isLocal should be removed in the future

Reply:
As i understand, this config is the configuration for something that will be pre-installed when init project. it's ok to leave anything included in this project

Reply:
What I meant is moving pipboard/daemon to init script instead of configuration.